content/efcore/getting-started/samples/manipulating-entities/adding-related-entities/Program.cs

```diff
@@ -28,9 +29,11 @@ public static void Main()
             var addedAuthor = context.Authors
                 .Include(a => a.Books)
                 .Single(a => a.LastName.Contains("Shelley"));
+				
+			Console.WriteLine("We added the author {0} {1} with their book(s) -", addedAuthor.FirstName, addedAuthor.LastName);
```

Comment:
Colons are used to introduce lists, not dashes:

- http://www.wheaton.edu/Academics/Services/Writing-Center/Writing-Resources/Semicolons-Colons-and-Dashes
- http://writingcenter.unc.edu/handouts/semi-colons-colons-and-dashes/
- https://www.pugetsound.edu/academics/academic-resources/cwlt/writing-advisor-schedule/writing-resources/colon-and-dashes/
- https://www.bethel.edu/undergrad/academics/support/colon-dash.pdf